src/main/kotlin/next/step/calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,29 @@
+package next.step.calculator
+
+object StringAddCalculator {
+
+    private val CUSTOM_COMMAND_PATTERN_REGEX = Regex("//(.)\n(.*)")
+    private val DEFAULT_COMMAND_PATTERN_REGEX = "[,:]".toRegex()
+
+    fun add(command: String?): Int {
+        if (command.isNullOrBlank()) {
+            return 0
+        }
+        return parse(command).sumOf { toNonNegativeInt(it) }
+    }
```

Comment:
코틀린에서 if는 "문"이 아니라 "식"이어서 아래 처럼도 작성해볼 수 있어요.
고칠 필요 없이 참고 정도만 하셔도 좋습니다 :)

```suggestion
    fun add(command: String?): Int {
        return if (command.isNullOrBlank()) {
            0
        } else { 
            parse(command).sumOf { toNonNegativeInt(it) }
        }
    }
```

Reply:
@malibinYun 피드백주신 내용 참고해서 아래와 같이 작성해보려 합니다. 아래와 같이 한 라인에 쓰는 걸 선호하시는지, 위와 같이 brace로 줄바꿈해서 작성하는 걸 선호하시는지 궁금합니다:)

```kotlin
if (command.isNullOrBlank()) 0 else parse(command).sumOf { toNonNegativeInt(it) }
```